### PR TITLE
Lighthouse Performance Fixes

### PR DESCRIPTION
**Pull request by Tarun-Arora**
## Fixes

Fixes #[issue number] by @[issue author]

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** guidelines in repository.
- [ ] My pull request has a descriptive title and is attached to an existing issue.
- [ ] I haven't repeated the code and it follows established code style of repository.
- [ ] My branch is up-to-date with upstream branch.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jyshdewangan** [comment]
- remove unused JS in aboutuslogo component

**Review by jyshdewangan — COMMENTED** [review]
- remove unused js in aboutuslogo component

File: components/Footer.vue [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <footer class="p-4 bg-white rounded-lg shadow md:px-8 container mx-auto">
     <hr class="my-2 border-gray-200 sm:mx-auto  lg:my-4">
     <div class="flex flex-col md:flex-row items-center md:justify-around">
-      <a href="https://copsiitbhu.co.in" target="_blank" class="flex items-center mb-4 sm:mb-0 order-1">
-        <nuxt-img src="/COPS-Logo.jpeg" width="80" height="80" alt="COPS Logo">
+      <a href="https://copsiitbhu.co.in" target="_blank" rel="noreferrer" class="flex items-center mb-4 sm:mb-0 order-1">
+        <nuxt-img src="/COPS-Logo.webp" width="80" height="80" alt="COPS Logo">
```
Comment (king-11) [inline components/Footer.vue:6]: no lazy loading

File: components/sdgCard.vue [hunk 2]
```diff
@@ -19,6 +19,7 @@
         <li class="inline-block">
           <a
             aria-label="linkedin"
+            target="_blank"
```
Comment (king-11) [inline components/sdgCard.vue:22]: no rel="no referrer"

**Review by king-11 — CHANGES_REQUESTED** [review]
- I see that some `a` have `rel` and target` set some don't we consistently need them for external links
- Many images don't have lazy loading yet a vscode search might do good

**Comment by king-11** [comment]
With [Nuxt Link](https://nuxtjs.org/docs/features/nuxt-components/#the-nuxtlink-component) we would have to be careful about smart prefetching that `nuxt-link` will do otherwise we might end up with too much javascript being loaded at once. For this we can:
- mark footer links with `no-prefetch`
- mark some heavy javascript pages with `no-prefetch`

**Review by king-11 — APPROVED** [review]

**Pull request merged**